jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/J2735GNSSstatus.java

```diff
@@ -1,22 +0,0 @@
-/*******************************************************************************
```

Comment:
Let's keep this class and turn it into GNSSStatus enum  and remove the definition from BitStringBuilder. Or define it  inside another relevant class.